blocksync/reactor.go

```diff
@@ -334,10 +365,14 @@ FOR_LOOP:
 			// routine.
 
 			// See if there are any blocks to sync.
-			first, second := bcR.pool.PeekTwoBlocks()
+			first, second, extCommit := bcR.pool.PeekTwoBlocks()
 			// bcR.Logger.Info("TrySync peeked", "first", first, "second", second)
```

Comment:
Can we remove this commented out code while we're here?

Reply:
This is done in the upcoming cherry-pick. Bear with me